README.md

```diff
@@ -384,7 +384,7 @@ $ ./deploy/k8s/deploy-local.sh
 $ logmein
 ```
 
-See the [deployment guide](#how-to-deploy) for further reading. To read more about the environment variables see the
+See the [Installation via Scripts](#installation-via-scripts) for further reading. To read more about the environment variables see the
```

Comment:
```suggestion
See [Installation via Scripts](#installation-via-scripts) for further reading. To read more about the environment variables, consult
```

Reply:
LGTM!
Nit: The suggestion reads a bit more fluent to me, but that's very optional